subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -126,6 +126,12 @@ config BT_CTLR_RX_STACK_SIZE
 	int
 	default 896
 
+config BT_CTLR_HCI_CMD_TX_COUNT
+	# Hidden Controller implementation supported Num_HCI_Command_Packets value */
+	int
+	depends on BT_CTLR_HCI
+	default 1
```

Comment:
Why can't we use `BT_BUF_CMD_TX_COUNT` instead?

Reply:
I should rename this to `BT_CTLR_HCI_NUM_CMD_PKT_MAX`, right? This is the supported 1 normal HCI command flow control count.

`BT_BUF_CMD_TX_COUNT` will be the buffer counts calculated with/without flow control support.

Reply:
A promptless Kconfig with a static value doesn't really have any value (i.e. nonconfigurable) - It could just as well be a `#define` or something.

Reply:
As the derived value `BT_BUF_CMD_TX_COUNT` in `hci_common_internal.h` is needed by `hci_raw.c` (and a Controller implementation HCI driver glue) which will be used with other vendor Controllers (source or binary) to build Controller only builds (hci_ipc, hci_uart, etc...), I do not want to hard code using `#define` which will require `hci_raw.c` to include numerous vendor Controller's header file. Kconfig is cleaner to fetch a Controller's implementation specifics (just like we have the numerous _SUPPORT Kconfigs).